src/pcapi/alembic/versions/20210629_f91ee203522a_add_educational_booking_model.py

```diff
@@ -0,0 +1,68 @@
+"""add_educational_booking_model
+
+Revision ID: f91ee203522a
+Revises: 3c46248f7143
+Create Date: 2021-06-29 10:43:18.269044
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "f91ee203522a"
+down_revision = "3c46248f7143"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "educational_booking",
+        sa.Column("id", sa.BigInteger(), autoincrement=True, nullable=False),
+        sa.Column("educationalInstitutionId", sa.BigInteger(), nullable=False),
+        sa.Column("educationalYearId", sa.String(30), nullable=False),
+        sa.Column("depositId", sa.BigInteger(), nullable=False),
+        sa.Column("status", sa.Enum("REFUSED", "USED_BY_INSTITUTE", name="educationalbookingstatus"), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["depositId"],
+            ["deposit.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["educationalInstitutionId"],
+            ["educational_institution.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["educationalYearId"],
+            ["educational_year.adageId"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(op.f("ix_educational_booking_depositId"), "educational_booking", ["depositId"], unique=False)
+    op.create_index(
+        op.f("ix_educational_booking_educationalInstitutionId"),
+        "educational_booking",
+        ["educationalInstitutionId"],
```

Comment:
```suggestion
        ["educationalInstitutionId", "educationalYearId"],
```

Il y a des chances que les deux soient utilisés en conjonction, et comme ça pas besoin de créer le troisième index `ix_educational_booking_educationalYearId`